inference-engine/tests/functional/plugin/cpu/test_utils/fusing_test_utils.hpp

```diff
@@ -0,0 +1,91 @@
+const auto fusingElu = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Elu, 2.0f)}}, {"Elu"}};
+const auto fusingSigmoid = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Sigmoid)}}, {"Sigmoid"}};
+const auto fusingClamp = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Clamp, 3.0f, 6.0f)}}, {"Clamp"}};
+const auto fusingPRelu = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::LeakyRelu), {{"Granularity" , "PerChannel"}}}}, {"PRelu"}};
+
+const auto fusingReluScaleShift = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Relu)},
+         {std::make_shared<ngraph::opset1::Multiply>(fakeConstNode, fakeConstNode), {{"Granularity" , "PerChannel"}}},
+         {std::make_shared<ngraph::opset1::Add>(fakeConstNode, fakeConstNode), {{"Granularity" , "PerChannel"}}}}, {"Relu", "Add"}};
+const auto fusingFakeQuantizePerChannelRelu = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeFakeQuantize(fakeConstNode, ngraph::element::f32, 256, fakeShape), {{"Granularity", "PerChannel"}}},
+         {ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Relu)}}, {"FakeQuantize", "Relu"}};
+// TODO: failed test
+// const auto fusingFakeQuantizePerTensorRelu = fusingSpecificParams{nullptr,
+//         {{ngraph::builder::makeFakeQuantize(fakeConstNode, ngraph::element::f32, 256, fakeShape), {{"Granularity", "PerTensor"}}},
+//          {ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Relu)}}, {"FakeQuantize", "Relu"}};
+const auto fusingSum = fusingSpecificParams{nullptr,
+        {{std::make_shared<ngraph::opset1::Add>(fakeConstNode, fakeConstNode), {{"Inputs", "Parameters"}}}}, {"Add"}};
+// todo: DWConvolution (not supported for GroupConvolution)
+
+/* "HARD" FUSING PATTERNS */
+std::shared_ptr<ngraph::Function> makeSwishPattern();
+const auto fusingSwishPattern = fusingSpecificParams{makeSwishPattern(), {}, {"Swish"}};
+
+std::string postNodes2str(const std::vector<postNode> &postNodes);
+std::shared_ptr<ngraph::Function> makeNgraphFunction(const ngraph::element::Type &ngPrc, ngraph::ParameterVector &params,
+        const std::shared_ptr<ngraph::Node> &lastNode, const std::shared_ptr<ngraph::Function> &postFunction);
+std::shared_ptr<ngraph::Function> makeNgraphFunction(const ngraph::element::Type &ngPrc, ngraph::ParameterVector &params,
+        const std::shared_ptr<ngraph::Node> &lastNode, const std::vector<postNode> &postNodes);
+
+void inline CheckFusing(InferenceEngine::ExecutableNetwork &execNet, std::string nodeType, std::vector<std::string> fusedOps) {
```

Comment:
Is it possible the situation that we will have tests in which will be more than one same type of node? (to cover bugs, for example)